### DISCUSSION TITLE
Timesketch Codebase

### DISCUSSION THREAD

**Discussion by sahasush** [google/timesketch#2941] [General]
Hi
I created a codebase and am trying to edit some changes but its not reflecting in my codebase environment.
I followed the steps from here:https://timesketch.org/developers/getting-started/

 made changes to these files to test
timesketch -->frontend-ng-->public -->index,html

But when I started ./tsdev.sh web, I dont see the changes ?
Anyone used codebase and how did you edit and make changes ?

**Comment by jkppr** [comment]
For frontend changes to take effect you need to run the frontend dev server as well. See https://timesketch.org/developers/getting-started/#frontend-ng-ui-development

This server will listen on port 5001 and changes will only be visible there until build.
